test/testng.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Suite">
+  <test name="Test">
+    <classes>
+      <class name=".HandBack"/>
+      <class name=".TestGCPolicyNogc"/>
+    </classes>
+  </test> <!-- Test -->
+</suite> <!-- Suite -->
```

Comment:
wrong change. 
expected to touch one of module.xml to allow include from gc_base to gc_modron_standard